<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Note</class>
 <widget class="QDialog" name="Note">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NOTES</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cne">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_idmat">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_note">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>CNE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID matiere</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ajoutnot">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modifnot">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supnote">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_CNE">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ID">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_affnoteleve">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher les notes d'eleve</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_affnotemat">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher les notes dans une matiere</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_note">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>20</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nommat">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de la matiere</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rafraichir">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
